worker/Dockerfile.template

```diff
@@ -16,30 +17,30 @@ COPY bin bin
 
 RUN npm run build
 
-FROM balenalib/%%BALENA_ARCH%%-node:12-bullseye-20211015
+FROM balenalib/%%BALENA_ARCH%%-alpine-node:12.19.1-3.12-run-20201211
 
-ENV UDEV=1
+ENV UDEV 1
 
-RUN install_packages \
-  libusb-1.0.0-dev libdbus-1-dev \
-  gstreamer1.0-tools gstreamer1.0-plugins-base gstreamer1.0-plugins-bad gstreamer1.0-plugins-good \
-  qemu-system-x86 ovmf bridge-utils iproute2 dnsmasq iptables ebtables qemu-utils
-
-WORKDIR /usr/app
-
-COPY config config
-COPY entry.sh .
+# hadolint ignore=DL3018
+RUN apk add --no-cache \
+  libusb-dev dbus-dev \
+  gstreamer-tools gst-plugins-base gst-plugins-bad gst-plugins-good \
+  bridge bridge-utils iproute2 dnsmasq iptables qemu-img qemu-system-x86_64 qemu-modules && \
+  apk add --arch x86_64 --root / --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.12/community ovmf
```

Comment:
OVMF only has published packages for x86_64 and aarch64 so explicitly install the x86_64 version for now.